src/vs/base/browser/markdownRenderer.ts

```diff
@@ -850,10 +834,6 @@ function fillInIncompleteTokensOnce(tokens: marked.TokensList): marked.TokensLis
 	return null;
 }
 
-function completeCodeBlock(tokens: marked.Token[], leader: string): marked.Token[] {
```

Comment:
The new version of marked does this automatically. However it does not close the other elements automatically so we can't remove all of `fillInIncompleteTokens`

Reply:
Nice, if it did it for all elements automatically I would be so annoyed haha